client/my-sites/earn/home.tsx

```diff
@@ -212,6 +213,65 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		};
 	};
 
+	/**
+	 * Return the content to display in the Donations card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getDonationsCard = () => {
+		const cta = isFreePlan
+			? {
+					text: translate( 'Unlock this feature' ),
+					action: () => {
+						trackUpgrade( 'any-paid-plan', 'donations' );
+						page( `/plans/${ selectedSiteSlug }` );
+					},
+			  }
+			: {
+					text: translate( 'Learn more about donations' ),
+					action: () => {
+						trackCtaButton( 'donations' );
+						if ( window && window.location ) {
+							window.location.href = localizeUrl( 'https://en.support.wordpress.com/donations/' );
+						}
+					},
```

Comment:
This can be simplified:
```suggestion
					action: { url: localizeUrl( 'https://wordpress.com/support/donations/' ), onClick: () => trackCtaButton( 'donations' ) },
```